types/imperium/index.d.ts

```diff
@@ -0,0 +1,98 @@
+// Type definitions for imperium 2.0
+// Project: https://www.npmjs.org/package/imperium
+// Definitions by: Gaetan SENN <https://github.com/gaetansenn>
+// Definitions: https://github.com/psnider/DefinitelyTyped/imperium
+// TypeScript Version: 2.7
+
+/// <reference types="node" />
+
+import express = require('express');
+
+type GetAcl = (req: express.Request) => Promise<boolean> | Promise<object>;
```

Comment:
Please fix the lint errors by adding `export` to these unexported types and interfaces.

Reply:
```
ERROR: 11:6   strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 12:6   strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 13:6   strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 15:11  strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 65:11  strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 69:11  strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 74:11  strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
```